doc/prototype/difference.md

```diff
@@ -6,25 +6,28 @@ in the another set.
 
 @signature `queryLogic.difference(a, b)`
 
-Returns a set that represents the difference of sets _A_ and _B_. In set theory, a difference is
-represented by `A \ B`.
-
-```js
-var queryLogic = new QueryLogic();
-
-
-queryLogic.difference({
-    filter: { name: {$in: ["Ramiya", "Bohdi"]} }
-},{
-    filter: { name: "Bohdi" }
-}))
-//-> {filter: {name: "Ramiya"}}
-
-// A is totally inside B
-queryLogic.difference(
-    {filter: { name: "Bohdi" }},
-    {} )  //-> QueryLogic.EMPTY
-```
+  Returns a set that represents the difference of sets _A_ and _B_. In set theory, a difference is
+  represented by `A \ B`.
+
+  ```js
+  import {QueryLogic} from "can";
+
+  const queryLogic = new QueryLogic();
+
+  const filter = queryLogic.difference(
+    { filter: { name: {$in: ["Ramiya", "Bohdi"]} } },
+    { filter: { name: "Bohdi" } }
+  );
+  console.log( filter ); //-> {filter: {name: "Ramiya"}}
+
+  // A is totally inside B
+  const emptyFilter = queryLogic.difference(
+    { filter: { name: "Bohdi" } },
+    {}
+  );
+  console.log( emptyFilter ); //-> QueryLogic.EMPTY
+  ```
+  @codepen
 
   @param  {can-query-logic/query} a A query.
```

Comment:
We should say something about how we are going to return a set representing the values in `a` that are not in `b` with respect to these params.  Might be worth looking at what these operands are called in set theory...

Reply:
a .. A query representing a set of data.  The `a` query can be looked at as having the `b` values removed to produce the result.